lang/MoarVM/Portfile

```diff
@@ -100,3 +89,14 @@ if {${os.platform} eq "darwin" && ${os.major} <= 10} {
 universal_variant   no
 # Unsupported by Configure.pl
 # configure.universal_args-delete --disable-dependency-tracking
+
+variant macports_libuv description "Use Macports libuv" {
+    depends_lib-append  port:libuv
+
+    configure.args-append \
+                    --has-libuv
+}
+
+# This can be enabled once libuv port is updated.
+# See: https://trac.macports.org/ticket/68464
+# default_variants-append macports_libuv
```

Comment:
so if someone would select this variant right now, does that work or not? If not, the variant should not be included yet but only after the update of libuv

Reply:
Yes, it will, but only with `libuv-devel`.

Reply:
okay, so it will not work as-is; that should be fixed by specifying the -devel dependency here for now. That can be changed to the libuv port accompanied by setting it as default variant once the libuv port is updated and it seems to work then as you have it here.

Reply:
Sure, I can do this.